js/repl/src/CodeMirror.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+import { css } from 'glamor';
+import React from 'react';
+import { colors } from './styles';
+
+const DEFAULT_CODE_MIRROR_OPTIONS = {
+  autoCloseBrackets: true,
+  keyMap: 'sublime',
+  matchBrackets: true,
+  mode: 'text/jsx',
+  lineNumbers: true,
```

Comment:
Nit: Alphabetize? This list of options is *almost* alphabetical, except for `lineNumbers` being in the wrong spot. 😛